solidity-checks/src/main/java/org/sonarsource/solidity/checks/CheckVerifier.java

```diff
@@ -18,6 +21,22 @@
   private static final Logger LOG = Loggers.get(CheckVerifier.class);
   private static final Integer SUFFIX_LENGTH = 2;
 
+  public static ParseTreePatternMatcher eCheckVerifier(boolean treePattern, String relativePath) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused method parameter "treePattern". ([squid:S1172](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS1172))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=35&issues=AWR-TYhuI66nRh8YLRI4&open=AWR-TYhuI66nRh8YLRI4)

[ISSUE_KEY]: AWR-TYhuI66nRh8YLRI4